app/namespace.go

```diff
@@ -668,6 +679,69 @@ func NewNamespaceCommand(getNamespaceClientFn GetNamespaceClientFn) (CommandOut,
 				return c.addRegion(ctx)
 			},
 		},
+		{
+			Name:    "delete-protection",
+			Usage:   "Enable delete protection on a temporal namespace",
+			Aliases: []string{"dp"},
+			Flags: []cli.Flag{
+				RequestIDFlag,
+				ResourceVersionFlag,
+				&cli.StringFlag{
+					Name:     NamespaceFlagName,
+					Usage:    "The namespace hosted on temporal cloud",
+					Aliases:  []string{"n"},
+					Required: true,
+				},
+				&cli.BoolFlag{
+					Name:    enableDeleteProtectionFlagName,
+					Usage:   "Enable delete protection on the namespace",
+					Aliases: []string{"edp"},
+				},
+				&cli.BoolFlag{
+					Name:    disableDeleteProtectionFlagName,
+					Usage:   "Enable delete protection on the namespace",
+					Aliases: []string{"ddp"},
+				},
+			},
+			Action: func(ctx *cli.Context) error {
```

Comment:
suggestion: we don't we follow the pattern for `auth-method` where we have a top-level NS command of `delete-protection` and two subcommands: `get` and `set`. 

https://github.com/temporalio/tcld/blob/8c55f7a12e093aed46517df0ebb90ccbafdfa3d7/app/namespace.go#L870.

Reply:
I can add set / get.

I single flag though is a little confusing for updates because ommitting the flag is false. We would need to use strings to represent true or false.

Reply:
you can make the CLI flag required.